### ISSUE TITLE
`GLIBC_2.33' not found

### ISSUE THREAD

**Issue by jecs** [ogri-la/strongbox#260]
Hi there.

I'm running into the following issue when running the latest strongbox executable from Releases: 
```
$ strongbox 
./usr/bin/java: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_strongHJWheK/usr/bin/../lib/jli/libjli.so)
```
I believe that this can be caused by compiling using a newer glibc. Unfortunately a lot of LTS releases are still on 2.32. Any chance you'd be willing to compile using glibc-2.32 or should I consider upgrading my distro?

**Comment by torkus** [comment]
hey @jecs (and @Invalidnam3 ;), thanks for giving strongbox a go and opening a ticket, I had no idea it was failing on an LTS release.

Without any details to go on I'm going to assume you're on a Ubuntu LTS, either 18.04 or 20.04 running a regular DE like Gnome or KDE. I've been able to replicate the problem on Ubuntu 20.04.

First off, strongbox isn't compiling like you might think. It *is* bundled into a single AppImage distributable and I think there is some extra magic I could be doing to make this work for you. While I figure that out, could you let me know if the `.jar` version works for you? You'll need Java 11 installed. The package name is the same in the latest two LTS releases:

    # apt install openjdk-11-jre
    $ java -jar strongbox-4.1.0-standalone.jar

I've confirmed this works on Ubuntu 20.04.

If you're still having problems, confirm you're using the right version of java with `java -version`.

You can switch between installations with `# update-alternatives --config java`

**Comment by Invalidnam3** [comment]
I'm able to use the java standalone client with no issue, I had the problem in Pop OS 20.10 which is Ubuntu based.

**Comment by torkus** [comment]
I've uploaded a version of strongbox built on an Ubuntu 18.04 machine to the [4.1.0 release](https://github.com/ogri-la/strongbox/releases/tag/4.1.0):

https://github.com/ogri-la/strongbox/releases/download/4.1.0/strongbox-4.1.0-2
https://github.com/ogri-la/strongbox/releases/download/4.1.0/strongbox-4.1.0-2.sha256

Could you give it a shot and let me know if it works for you guys?

**Comment by Invalidnam3** [comment]
The AppImage works for me now, thanks for the help :+1:

**Comment by jecs** [comment]
@torkus, you were right on the money. I am on Ubuntu 20.04. And the binary that you uploaded works like a charm. Thank you very much!

**Comment by Lanecc-CITLab** [comment]
I can tell you it doesn't work on OpenSUSE Leap 15.2 with the same error.

**Comment by jecs** [comment]
> I can tell you it doesn't work on OpenSUSE Leap 15.2 with the same error.

Did you try the new executable under Releases? [https://github.com/ogri-la/strongbox/releases/download/4.1.0/strongbox-4.1.0-2](https://github.com/ogri-la/strongbox/releases/download/4.1.0/strongbox-4.1.0-2)

**Comment by torkus** [comment]
thanks guys, much appreciated.

> I can tell you it doesn't work on OpenSUSE Leap 15.2 with the same error.

@Lanecc-CITLab , could you double check which version you used for me?

**Comment by torkus** [comment]
`4.2.0` has just been released and was built on Ubuntu 18.04 like the `4.1.0-2` release that seemed to work for people.

If anyone finds they have a similar problem in future, please open a ticket with the name and version of your distribution. I won't know it's breaking otherwise.

Thanks everyone for their help, closing this.
